app/kuma-dp/cmd/run_test.go

```diff
@@ -125,12 +118,11 @@ var _ = Describe("run", func() {
 			// then
 			var actualConfigFile string
 			envoyPid := verifyComponentProcess("Envoy", envoyPidFile, envoyCmdlineFile, func(actualArgs []string) {
-				Expect(actualArgs[0]).To(Equal("--version"))
```

Comment:
Why is the version no longer there? I assume it was not needed 🤔

Reply:
That’s because version retrieval was moved to the `BootstrapClient`, and in these tests, we’re using a fake client that doesn’t perform that logic.